src/client/components/footer/Footer.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import styles from './Footer.module.css'
+import './Footer.module.css';
 
 const Footer = (props) => {
   return (
-    <div className={styles.footer}>
-      <span id = {styles.footerText}> {props.footerText} </span>
+    <div className="footer">
+      <span>NETFLIX</span>ROULETTE
```

Comment:
Why NETFLIX and ROULETTE is hardcoded. Let's pass it from parent components

`<span>NETFLIX ROULETTE</span>`

Reply:
Done)